LogicalTypes.md

```diff
@@ -144,6 +144,18 @@ example, there is no requirement that a large number of days should be
 expressed as a mix of months and days because there is not a constant
 conversion from days to months.
 
+### INT96 timestamps (also called IMPALA_TIMESTAMP)
+
+_(deprecated)_ Timestamps saved as an `int96` are made up of the nanoseconds in the day
```

Comment:
Just a suggestion from a random passer by... I think it might be nice to document the byte order of these timestamps as well, as they appear to be little endian whereas other primitive ints in parquet are big endian. At least that seems to be the issue that's lead us to finding this PR in the first place :)